sdk/core/azure-core/README.md

```diff
@@ -1,6 +1,6 @@
 # Azure Core client library for Java
 
-[![Build Documentation](https://img.shields.io/badge/documentation-published-blue.svg)](https://azure.github.io/azure-sdk-for-java/track2reports/index.html)
+[![Build Documentation](https://img.shields.io/badge/documentation-published-blue.svg)](https://azure.github.io/azure-sdk-for-java)
```

Comment:
quick comment --- the new landing page needs improvements - "Latest SDK" and "Previous SDK" for "Track 2" and "Track 1" isn't very clear for customers.  @kaerm is this a doc asset that you're working on?

Reply:
I am not. Also, unfortunately I do not think there is a better way for us to refer to them as we are not supposed to have a name or even use tracks. (I personally fully agree with the sentiment though) @joshfree

Reply:
@joshfree what needs to get updated is that these, will ultimately, should probably point to the same library that the readme is for (see @weshaggard's comment above). The doc index hasn't been updated (it has to wait, there's a change that needs to go into pom.client.xml and pom.data.xml those need to wait until the release is finished) and that landing page is broken up by area (keyvault, core, cosmos etc) which contains the list of libraries in that area both tracks. I have my own github.io updated and would be happy to show you what the docs are going to look like.